README.md

```diff
@@ -70,28 +70,6 @@ Example
 
 You can find the keyword documentation [here](https://raw.githack.com/Altran-PT-GDC/Robot-Framework-Mainframe-3270-Library/master/doc/Mainframe3270.html).
 
-## Importing
```

Comment:
This section is already included in the keyword documentation, I think it's best not to repeat it.